corehq/apps/app_manager/templates/app_manager/partials/forms/custom_assertions.html

```diff
@@ -1,7 +1,6 @@
 {% load i18n %}
 {% load hq_shared_tags %}
 {% initial_page_data 'custom_assertions' custom_assertions %}
-<script src="{% static 'app_manager/js/custom_assertions.js' %}"></script>
```

Comment:
Is it possible to use `{% block js %}{{ block.super }}...{% endblock %}` in a partial?